sonic_platform_base/sonic_xcvr/sfp_optoe_base.py

```diff
@@ -51,6 +51,14 @@ def get_transceiver_pm(self):
         api = self.get_xcvr_api()
         return api.get_transceiver_pm() if api is not None else None
 
+    def freeze_vdm_stats(self):
+        api = self.get_xcvr_api()
+        return api.freeze_vdm_stats() if api is not None else None
+
+    def unfreeze_vdm_stats(self):
```

Comment:
@jaganbal-a can you add comment for return value status for True(Success) and False(Failure)